connect/runtime/src/test/java/org/apache/kafka/connect/integration/OffsetsApiIntegrationTest.java

```diff
@@ -767,11 +775,7 @@ public void testResetSinkConnectorOffsetsZombieSinkTasks() throws Exception {
             connect.kafka().produce(TOPIC, 0, "key", "value");
         }
 
-        // Configure a sink connector whose sink task blocks in its stop method
-        Map<String, String> connectorConfigs = new HashMap<>();
-        connectorConfigs.put(CONNECTOR_CLASS_CONFIG, BlockingConnectorTest.BlockingSinkConnector.class.getName());
-        connectorConfigs.put(TOPICS_CONFIG, TOPIC);
-        connectorConfigs.put("block", "Task::stop");
+        Map<String, String> connectorConfigs = baseSinkConnectorConfigs();
```

Comment:
Same as above, `baseSinkConnectorConfigs()` can be used directly in the call to `configureConnector`